deps/obs-scripting/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ target_sources(
   PUBLIC obs-scripting.h
   PRIVATE obs-scripting-callback.h obs-scripting-logging.c obs-scripting.c)
 
-target_compile_definitions(obs-scripting PRIVATE SCRIPT_DIR="${OBS_SCRIPT_PLUGIN_PATH}"
+target_compile_definitions(obs-scripting PRIVATE SCRIPT_DIR="../${OBS_SCRIPT_PLUGIN_PATH}"
```

Comment:
Has this been tested on Windows? The variable is empty on macOS, but is set to `../../${OBS_DATA_DESTINATION}/obs-scripting/64bit` on Windows, so I'd be surprised if this didn't break.

If this is change universally correct on Linux, FreeBSD, and OpenBSD, then this change should be put into `defaults.cmake` for Linux.

Reply:
:facepalm:

Reply:
Should be better now.